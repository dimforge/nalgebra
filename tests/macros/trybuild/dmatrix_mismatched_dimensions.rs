use nalgebra::dmatrix;

fn main() {
    dmatrix![1, 2, 3;
            4, 5];
}