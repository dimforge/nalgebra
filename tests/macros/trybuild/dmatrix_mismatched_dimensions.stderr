error: Unexpected number of entries in row 1. Expected 3, found 2 entries.
 --> $DIR/dmatrix_mismatched_dimensions.rs:5:13
  |
5 |             4, 5];
  |             ^
